core/templates/domain/collection/collection-node.model.ts

```diff
@@ -43,13 +43,6 @@ export class CollectionNode {
     return new CollectionNode(collectionNodeBackendObject);
   }
 
-  static createFromExplorationId(explorationId: string): CollectionNode {
-    return CollectionNode.create({
-      exploration_id: explorationId,
-      exploration_summary: null
-    });
-  }
```

Comment:
Why remove this function? Did you refactor all the places that were using this function?

Reply:
This function was failing the strict check because of the null assignment to the exploration summary. It seems like this function wasn't necessary so I refactored the code where it was being used. Most of the uses were in test files.